src/lib/optimizer/chunk_statistics/chunk_column_statistics.cpp

```diff
@@ -21,14 +25,15 @@ static std::shared_ptr<ChunkColumnStatistics> build_statistics_from_dictionary(c
   auto statistics = std::make_shared<ChunkColumnStatistics>();
   // only create statistics when the compressed dictionary is not empty
   if (!dictionary.empty()) {
-    auto min_max_filter = std::make_unique<MinMaxFilter<T>>(dictionary.front(), dictionary.back());
-    statistics->add_filter(std::move(min_max_filter));
-
     // no range filter for strings
```

Comment:
I am fine with that solution, but wouldn't it be possible to determine min and max strings?

Reply:
Not for the gap filter, though. I guess calculating distances for strings might be too much.